packages/scss/src/components/formLabel/component.scss

```diff
@@ -1,21 +1,30 @@
 @mixin component($atRoot: 'without: rule') {
-  color: var(--components-formLabel-color);
-  display: inline-block;
-  font-size: var(--components-formLabel-fontSize);
-  line-height: var(--components-formLabel-lineHeight);
-  width: fit-content;
+	color: var(--components-formLabel-color);
+	display: flex;
+	font-size: var(--components-formLabel-fontSize);
+	line-height: var(--components-formLabel-lineHeight);
+	width: var(--components-formLabel-width);
+	gap: var(--spacings-XS);
```

Comment:
This gap generate spacing issues with tootltip "?" icon and mandatory's "*".
<img width="115" alt="image" src="https://github.com/LuccaSA/lucca-front/assets/25581936/49cd8682-99a9-4aba-95bc-edd07399ec78">